Index: zotero-standalone-build/assets/prefs.js
===================================================================
--- zotero-standalone-build.orig/assets/prefs.js	2013-03-28 17:39:36.000000000 +0100
+++ zotero-standalone-build/assets/prefs.js	2013-03-28 17:39:36.000000000 +0100
@@ -77,7 +77,7 @@
 
 /** The below is imported from https://developer.mozilla.org/en/XULRunner/Application_Update **/
 // Whether or not app updates are enabled
-pref("app.update.enabled", true);
+pref("app.update.enabled", false);
 
 // This preference turns on app.update.mode and allows automatic download and
 // install to take place. We use a separate boolean toggle for this to make
@@ -145,4 +145,4 @@
 pref("app.update.incompatible.mode", 0);
 
 // update channel for this build
-pref("app.update.channel", "default");
\ No newline at end of file
+pref("app.update.channel", "default");
Index: zotero-standalone-build/modules/zotero/chrome/content/zotero/standalone/standalone.xul
===================================================================
--- zotero-standalone-build.orig/modules/zotero/chrome/content/zotero/standalone/standalone.xul	2013-03-28 17:39:36.000000000 +0100
+++ zotero-standalone-build/modules/zotero/chrome/content/zotero/standalone/standalone.xul	2013-03-28 17:39:36.000000000 +0100
@@ -197,10 +197,12 @@
 									  accesskey="&helpReportErrors.accesskey;"
 									  label="&helpReportErrors.label;"
 									  command="cmd_zotero_reportErrors"/>
+							<!--
 							<menuitem id="checkForUpdates"
 									  accesskey="&helpCheckForUpdates.accesskey;"
 									  label="&helpCheckForUpdates.label;"
 									  oncommand="ZoteroStandalone.checkForUpdates();"/>
+							-->
 							<menuitem id="aboutName"
 									  accesskey="&aboutProduct.accesskey;"
 									  label="&aboutProduct.label;"
