Index: zotero-standalone-build/modules/zotero-libreoffice-integration/resource/installer.jsm
===================================================================
--- zotero-standalone-build.orig/modules/zotero-libreoffice-integration/resource/installer.jsm	2013-04-01 21:35:13.000000000 +0200
+++ zotero-standalone-build/modules/zotero-libreoffice-integration/resource/installer.jsm	2013-04-01 21:35:13.000000000 +0200
@@ -191,13 +191,19 @@
 				if(success) {
 					zpi.success();
 				} else {
-					openInstallationWizard();
+/*					openInstallationWizard();
+*/
+                                        zpi.success();
+
 				}
 			});
 		} else if(!zpi.failSilently || this.getPotentialInstallations().length) {
 			// otherwise, if there are installations and we are not failing silently, open the
 			// wizard
-			openInstallationWizard();
+/*			openInstallationWizard();
+*/
+                                        zpi.success();
+
 		}
 	}
 	
